app/assets/stylesheets/custom.css.scss

```diff
@@ -99,6 +99,45 @@ body {
   margin-top: 20px;
 }
 
+.stats {
+  @include span-columns(12);
+  @include omega();
+  $base-spacing: 1.5em !default;
+  padding: base-spacing /Z;
+  margin-top: 100px;
+  text-align: center;
+
+  ul {
+    padding: 0;
+  }
+
+  li {
+    border-right: 1px solid transparentize($dark-blue, 0.8);
+    color: $blue;
+    display: inline;
+    float: left;
+    font-size: 1.2em;
+    line-height: 1.1em;
+    padding: 0 0.7em;
+
+    &:first-child {
+      padding-left: 0;
+    }
+
+    &:last-child {
+      border-right: 0;
+    }
+  }
+
+  span {
+    color: $dark-blue;
+    display: block;
+    font-size: 1em;
+    font-weight: normal;
+  }
+}
+
+
 footer {
 
 }
```

Comment:
github is complaining that there is no newline at the end of this file. In sublime you can set `ensure_newline_at_eof_on_save` to make sure it always adds one if you forget. You don't need to know why this is preferable, but if your interested https://robots.thoughtbot.com/no-newline-at-end-of-file